prow/plugins.yaml

```diff
@@ -287,6 +287,15 @@ require_matching_label:
   prs: true
   regexp: ^(sig|wg|committee)/
   # Define needs-sig separately for issues so that we only comment on issues.
+- missing_label: needs-sig-triage
+  org: kubernetes
+  repo: kubernetes
+  issues: true
+  regexp: ^(needs-sig$)/
```

Comment:
What are you trying to match? It looks like this is trying to match a `/` after the end of the string (`$`)